psydac/api/grid.py

```diff
@@ -54,43 +77,6 @@ def weights(self):
     def quad_order(self):
         return [w.shape[1] for w in self.weights]
 
-#==============================================================================
-class BoundaryQuadratureGrid(QuadratureGrid):
```

Comment:
We don't need a `BoundaryQuadratureGrid` class any more?

Please explain, and add a few words to the PR's description,

Reply:
yes, there is no point in having two classes one is enough to prepare the data so that we can expose it to the user later

Reply:
Thanks. Please add this explanation to the PR description.

Reply:
done